home.html

```diff
@@ -20,9 +20,9 @@
   <body>
     <section class="header">
       <nav>
-        <a class="logo" href="home.html"
+        <!-- <a class="logo" href="home.html"
```

Comment:
@AmaniLouendriz if you're going to comment something, you may as well take it out completely. You can always look in source control if you want to get it back.